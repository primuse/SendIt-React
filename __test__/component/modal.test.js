import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import { MemoryRouter } from 'react-router-dom';
import configureStore from 'redux-mock-store';
import { Provider } from 'react-redux';
import thunk from 'redux-thunk';
import Modal from '../../src/components/modal.jsx';

global.Headers = () => { };

const initialState = {};
const mockStore = configureStore([thunk]);
const store = mockStore(initialState);

const form = () => <div></div>;


describe('All users component', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<MemoryRouter>
        <Provider store={store}>
          <Modal modalTitle={'New Parcel'} ModalForm={form} />
        </Provider>
      </MemoryRouter>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
