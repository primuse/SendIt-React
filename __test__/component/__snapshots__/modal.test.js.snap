// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`All users component renders correctly 1`] = `
<div
  className="modal"
>
  <div
    className="modal-box"
  >
    <div
      className="modal-content"
    >
      <h2
        className="text-center"
      >
        New Parcel
      </h2>
      <div />
    </div>
  </div>
</div>
`;
